src/main/java/org/ingress/cartms/client/AuthClient.java

```diff
@@ -0,0 +1,18 @@
+package org.ingress.cartms.client;
+
+import org.ingress.cartms.client.decoder.CustomErrorDecoder;
+import org.springframework.cloud.openfeign.FeignClient;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestHeader;
+
+import static org.springframework.http.HttpHeaders.AUTHORIZATION;
+
+@FeignClient(name= "ms-auth",
+        url = "${client.ms-auth.url}",
+        path = "/v1/verify", configuration = CustomErrorDecoder.class)
```

Comment:
add /internal

[Optional]: 
move API path (versioning/domain/operation) to http method, ex: `@PostMapping("v1/auth/verify")`